uefi/section_test.go

```diff
@@ -12,7 +12,7 @@ var (
 	emptySec     = make([]byte, 0)                                                     // Empty section
 	tinySec      = []byte{4, 0, 0, byte(SectionTypeRaw)}                               // Section header with no data
 	wrongSizeSec = append([]byte{40, 0, 0, byte(SectionTypeRaw)}, make([]byte, 20)...) // Section with a size mismatch
```

Comment:
Can you make a test like this one, but the actual size is too big?

Reply:
That's not an error but I can make one and check that an error isn't returned

Reply:
Done